README.md

```diff
@@ -217,6 +217,79 @@ The object returned from a call to `cordova.InAppBrowser.open`.
 
 - __callback__: the function that executes when the event fires. The function is passed an `InAppBrowserEvent` object as a parameter.
 
+## Example
+
+```javascript
+
+var inAppBrowserRef = undefined;
```

Comment:
Yep. I can do this by moving the other functions into the scope of the parent function